Python pokemon/generate_team.py

```diff
@@ -0,0 +1,46 @@
+import requests
+import random
+from pydantic import BaseModel
+import json
+
+class Pokemon(BaseModel):
+    name: str
+    types: list[str]
+    hp_stat: int
+    ability_names: list[str]
+
+team = []
+
+for _ in range(6):
+    response = requests.get(f"https://pokeapi.co/api/v2/pokemon/{random.randrange(1, 150)}/")
+    data = response.json()
```

Comment:
Du kunne eventuelt indsætte noget fejlhåndtering her, så du bedre kan håndtere hvis den for eksempel ikke kan hente en pokemon eller noget
```suggestion
    if response.status_code != 200:
        print(f"Error fetching data: {response.status_code}")
        exit(1)
    data = response.json()
```